elemental/client.py

```diff
@@ -7,7 +8,10 @@
 import xmltodict
 from jinja2 import Template
 
-TEMPLATE_PATH = "elemental/templates/qvbr_mediastore.xml"
+
+def file_fixture(file_name):
+    with open(os.path.join("elemental/templates", file_name)) as f:
+        return f.read()
```

Comment:
This `open` assumes you are running from the project folder, the path is relative, this is true now that you are running tests from that folder but it won't be the case when you have the app installed somewhere and import from another project.

Try this:
```python
# this code from client.py will give you the absolute path for folder elemental/,
# the python package we install (not the elemental folder from where you run your tests,
# with README, setup.py, etc)
folder = os.path.dirname(os.path.realpath(__file__))
os.path.join(folder, "templates", file_name)
```

there is a nice way to test this, you can install your package in your machine (or in a virtualenv) and `cd` into a random folder. Then `import elemental` and call the method (you can simple do a `print()` to test the path)

I like to do that `pip install -e .` from the project folder, this will install the current folder as "editable" using pip, which means if you edit the code your installed package will use it
```
  -e, --editable <path/url>   Install a project in editable mode (i.e. setuptools "develop mode") from a local project path or a VCS url.
```

then `cd /anywhere/in/your/computer`, open a `python` shell and try to use it. Make sure that function generates the same path no matter where you use your package from

Reply:
very helpful